coordinator/apps/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copy source code and build the project
+FROM node:20-alpine as builder
+
+WORKDIR /builder
+
+COPY . .
+
+RUN npm i -g pnpm@8
+RUN pnpm install --frozen-lockfile --prefer-offline
+RUN pnpm run build
+
+# Create image by copying build artifacts
+FROM node:20-alpine as runner
+RUN npm i -g pnpm@8
+
+RUN mkdir -p ~/rapidsnark/build; \
+    wget -qO ~/rapidsnark/build/prover https://maci-devops-zkeys.s3.ap-northeast-2.amazonaws.com/rapidsnark-linux-amd64-1c137; \
+    chmod +x ~/rapidsnark/build/prover
+RUN wget -qO ~/circom https://github.com/iden3/circom/releases/download/v2.1.6/circom-linux-amd64; \
+    chmod +x ~/circom; \
+    mv ~/circom /bin
+
+USER node
+ARG PORT=8545
+
+WORKDIR ./maci
+COPY --chown=node:node  --from=builder /builder/ ./
+WORKDIR /maci/coordinator
+
+EXPOSE ${PORT}
+CMD ["pnpm", "run", "hardhat", "--hostname", "0.0.0.0"]
```

Comment:
For this service it should be one more command `pnpm run start:prod`

Reply:
The command mentioned above, fails with the following output:
```
 
> maci-coordinator@0.1.0 start:prod /drill/coordinator
> node dist/main
 
node:internal/modules/cjs/loader:1148
  throw err;
  ^
 
Error: Cannot find module '/drill/coordinator/dist/main'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
    at Module._load (node:internal/modules/cjs/loader:986:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
 
Node.js v20.13.1
 ELIFECYCLE  Command failed with exit code 1.
```

Reply:
Got it, can you please change this command to `"start:prod": "node build/ts/main.js",`

Reply:
Done!